src/components/split-button/split-button.component.js

```diff
@@ -221,14 +222,17 @@ SplitButton.propTypes = {
   /** The text to be displayed in the SplitButton. */
   text: PropTypes.string.isRequired,
   /** Defines an Icon position within the button: "before" | "after" */
-  iconPosition: PropTypes.oneOf(OptionsHelper.buttonIconPositions)
+  iconPosition: PropTypes.oneOf(OptionsHelper.buttonIconPositions),
+  /** Set align of the rendered content */
+  align: PropTypes.string
```

Comment:
Should this be `PropTypes.oneOf(OptionsHelper.alignBinary)`?

Reply:
Good point! :) Fixed